README.md

```diff
@@ -13,7 +13,7 @@ It includes the [openui5-sample-app](https://github.com/SAP/openui5-sample-app)
 
 ## Requirements
 
-- [Node.js](https://nodejs.org/) Version 10 or later
+- [Node.js](https://nodejs.org/) Version 14 (npm version 8 is currently not supported)
```

Comment:
This project already uses npm v8 (see package-lock.json). Is there a reason why we shouldn't continue to?

Reply:
When a `npm install` on the root project is executed, there is a endless loop performing the `postinstall` hook. Somehow this does not happen when npm 6 is used.

Reply:
I guess that's because when using workspaces, you should only run "npm i" in the workspace use.
So either we should remove the workspaces config or (preferred) use the workspaces as intended, to only run "npm i" on root level and remove the package-lock.json in the sub-packages.

Reply:
@matz3 Got it. Thanks a lot for this hint!